tools/calculate-release/CalculateRelease.hs

```diff
@@ -48,8 +48,8 @@ main = do
 
     putStrLn $ "Mainnet fork height (at " <> show mainnetForkTime <> "): " <> show mainnetForkHeight
     putStrLn $ "Testnet fork height (at " <> show testnetForkTime <> "): " <> show testnetForkHeight
-    let nextServiceDateDay = addDays (8 * 7) serviceDateDay
-    putStrLn $ "Next service date (+8 weeks): " <> show nextServiceDateDay
+    let nextServiceDateDay = addDays (12 * 7) serviceDateDay
+    putStrLn $ "Next service date (+12 weeks): " <> show nextServiceDateDay
     where
```

Comment:
:+1:

Reply:
3859808 - 3805981 (current testnet height) = 53827 blocks.
53827 blocks / ~2 blocks per minute = 26914 minutes.
26914 minutes / 60 = 449 hours.
449 hours / 24 hours in a day = 18 and a bit days.
It's 18 and a bit days until Dec 14.